odoo/tools/mail.py

```diff
@@ -356,9 +356,9 @@ def html2plaintext(html, body_id=None, encoding='utf-8'):
     ## <peter@fry-it.com>
     ## download here: http://www.peterbe.com/plog/html2plaintext
 
-    html = ustr(html)
+    html = html and ustr(html)
```

Comment:
why not this, so we don't need to change the condition bellow
```suggestion
    html = ustr(html or '')
```